python/cudf/cudf/core/column/lists.py

```diff
@@ -46,6 +46,17 @@ def __init__(
         null_count=None,
         children=(),
     ):
+        if size is None:
```

Comment:
Can we do this handling of `size is None` at the `build_list_column` helper instead? I think it's cleaner to require the `size` here since it's a non keyword argument.

Reply:
sure, sounds good 👍

Reply:
Done